clang/test/Sema/vector-ast.c

```diff
@@ -0,0 +1,13 @@
+// RUN: %clang_cc1 %s -verify -ast-dump | FileCheck %s
+
+// expected-no-diagnostics
+
+// CHECK: VarDecl {{.*}} x 'int __attribute__((ext_vector_type(4)))'
```

Comment:
Oof, I hate the type printing here.  Nothing to change for that, but it makes this much less useful for ast-dump.

Do me a favor and do an ast-dump for:

`using ExtVecType = decltype(x);`

Which should eventually do something like:

```
`-ExtVectorType 'int __attribute__((ext_vector_type(4)))' 4
          `-BuiltinType 'int'
```